numba/core/inline_closurecall.py

```diff
@@ -1031,24 +1063,28 @@ def is_removed(val, removed):
     # Skip list construction and skip terminator, add the rest to stmts
     for i in range(len(loop_entry.body) - 1):
         stmt = loop_entry.body[i]
-        if isinstance(stmt, ir.Assign) and (stmt.value is list_def or is_removed(stmt.value, removed)):
+        if isinstance(stmt, ir.Assign) and (stmt.value is list_def
+                                            or is_removed(stmt.value, removed)):
             removed.append(stmt.target)
         else:
             stmts.append(stmt)
     debug_print("removed variables: ", removed)
 
     # Define an index_var to index the array.
-    # If the range happens to be single step ranges like range(n), or range(m, n),
-    # then the index_var correlates to iterator index; otherwise we'll have to
-    # define a new counter.
+    # If the range happens to be single step ranges like range(n), or range(m,
+    # n), then the index_var correlates to iterator index; otherwise we'll have
+    # to define a new counter.
```

Comment:
```suggestion
    # If the range happens to be single step ranges like range(n), or 
    # range(m, n), then the index_var correlates to iterator index; otherwise
    # we'll have to define a new counter.
```
keeps the range spelling on the same line.